Forwarded: not-needed

disable unrar test since the module is not in Debian main

---
 src/calibre/test_build.py |    5 +++++
 1 file changed, 5 insertions(+)

Index: b/src/calibre/test_build.py
===================================================================
--- a/src/calibre/test_build.py
+++ b/src/calibre/test_build.py
@@ -17,6 +17,10 @@ from polyglot.builtins import iteritems,
 
 is_ci = os.environ.get('CI', '').lower() == 'true'
 
+try:
+    import unrardll
+except ModuleNotFoundError:
+    unrardll = None
 
 class BuildTest(unittest.TestCase):
 
@@ -277,6 +281,7 @@ class BuildTest(unittest.TestCase):
         from calibre.gui2.win_file_dialogs import test
         test()
 
+    @unittest.skipUnless(unrardll, 'Module unrardll is missing')
     def test_unrar(self):
         from calibre.utils.unrar import test_basic
         test_basic()
